test/e2e/common/expansion.go

```diff
@@ -158,7 +158,7 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 		    Description: Make sure a container's subpath can be set using an
 			expansion of environment variables.
 	*/
-	ginkgo.It("should allow substituting values in a volume subpath [sig-storage]", func() {
+	framework.ConformanceIt("should allow substituting values in a volume subpath [sig-storage]", func() {
```

Comment:
Thanks for updating! I think we'll need to do this as a two-step process. 1) Update to emptydir. 2) Once we verify it's stable and not flaky, then promote to conformance.

Reply:
I have raised a separate Issue https://github.com/kubernetes/kubernetes/issues/87810 and will hold this one for the moment

Reply:
@msau42 Haven't seen any errors in sig testgrid since the HostPath change, so I would guess we're good to go with this one now

Reply:
I don't see anywhere obvious in the PR tests that has actually performed the conformance prior to merge. Is there any easy way of checking that this PR did some Conformance e2e?

Reply:
I found that this one `pull-kubernetes-conformance-kind-ipv6-parallel` has performed all 6 conformance tests, so that seems to confirm they are okay